### PR TITLE
Dummy

### PR DESCRIPTION
**Pull request by warquys**

### CODE REVIEW

**Comment by warquys** [comment]
The dummy can move, can be displayed in AR, I read back the player coming out of AR and do like the old dummy but remove player float rotation. I am unable to do the ShowPlayer().

I prepared events for the future in the dummyService, I could not test them and maybe they will have to be added in the codes of the futurs patch.

File: Synapse3.SynapseModule/Player/SynapsePlayerCommonProperties.cs [hunk 1]
```diff
@@ -247,6 +247,49 @@ public SynapsePlayer CurrentlySpectating
...
+    {
+        get => FirstPersonMovement?.CrouchSpeed ?? 0;
+        set
+        {
+            var firstperosn = FirstPersonMovement;
+            if (firstperosn != null)
+                firstperosn.CrouchSpeed = value;
+        }
+    }
+
```
Comment (GrafDimenzio) [inline Synapse3.SynapseModule/Player/SynapsePlayerCommonProperties.cs]: It only sets the speed server side and affects the anti cheat but doesn't sync with the client

Comment (warquys) [reply Synapse3.SynapseModule/Player/SynapsePlayerCommonProperties.cs]: yes but we can use it with a fake sync var, for the moment I don't know if it was already done so I didn't add it

Comment (warquys) [reply Synapse3.SynapseModule/Player/SynapsePlayerCommonProperties.cs]: ho the prop MovementMultiplier existe ok

**Comment by warquys** [comment]
I change it on read only, that can be use full to get the default speed of the role

**Comment by GrafDimenzio** [comment]
I don't think it's smart to add Rotation setters when they don't really work so I removed them and they will only be implemented back if we actually can set them

**Pull request merged**